### PR TITLE
feat: yarn run test in CI

### PR DESCRIPTION
**Pull request by keroro520**
Includes `yarn run test` in CI pipeline.

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -20,15 +20,42 @@ jobs:
...
+    - working-directory: godwoken-kicker
+      run: |
+        cat docker/layer2/config/web3-config.env | grep -v '^#'           >> $GITHUB_ENV
+        echo "PORT=9025"                                                  >> $GITHUB_ENV
+        echo "DATABASE_URL=postgres://user:password@127.0.0.1:5432/lumos" >> $GITHUB_ENV
+        echo "REDIS_URL=redis://127.0.0.1:6379"                           >> $GITHUB_ENV
+        echo "GODWOKEN_JSON_RPC=http://127.0.0.1:8119"                    >> $GITHUB_ENV
+        echo "GODWOKEN_WS_RPC_URL=ws://127.0.0.1:8120"                    >> $GITHUB_ENV
+    - name: Start godwoken-web3 service
+      run: yarn run start &
```
Comment (classicalliu) [inline .github/workflows/node.js.yml:58]: Using `yarn run build && yarn run start:prod &` or add a new command `yarn run start:dev` is better.
`yarn run build` will build all packages while `yarn run start` not, and no need to watch code changes here.

**Review by classicalliu — COMMENTED** [review]
I think move intergration test to a new action pipeline is better.

**Comment by keroro520** [comment]
> I think move intergration test to a new action pipeline is better.

Agree.
I will update it tomorrow.

**Comment by keroro520** [comment]
Replaced by https://github.com/nervosnetwork/godwoken-web3/pull/263

**Pull request closed without merging**